src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax;
+
+public abstract class Figure implements Area {
+    private String colour;
+    private String property;
```

Comment:
Let's remove field `property`

Reply:
If I remove the property field, how can I get the value of it in "drawFigure" method?